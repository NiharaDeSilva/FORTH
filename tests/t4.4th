2500 25 / 10 / .
1000 2.0 / CR .
12000.0 6 / CR .
23000 10 / CR .