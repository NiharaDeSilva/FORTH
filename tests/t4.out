DivisionIsFun!!!
10
500.0
2000.0
2300